### ISSUE TITLE
Groovy Cast Exception for NanoporeParser

### ISSUE THREAD

**Issue by sven1103** [qbicsoftware/java-openbis-dropboxes#17]
The `NanoporeParser` does not implement the `DatasetParser` interface yet, and therefore causes an unwanted GroovyCastException: 

```
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'life.qbic.utils.NanoporeParser@74d7ee84' with class 'life.qbic.utils.NanoporeParser' to class 'life.qbic.datasets.parsers.DatasetParser'
```

**Comment by sven1103** [comment]
Fix: remove the NanoporeParser support, there is no registry for it yet anyway

**Comment by sven1103** [comment]
Solved in #18
